app/controllers/course/survey/responses_controller.rb

```diff
@@ -77,7 +77,7 @@ def handle_create_error(error)
     @response = @survey.responses.accessible_by(current_ability).
                 find_by(course_user_id: current_course_user.id)
     if @response
-      render json: { responseId: @response.id }, status: :bad_request
+      render partial: 'created', status: :bad_request
```

Comment:
oops, I  actually meant 303, it means that the resources was created somewhere else.